src/core/ledger/applyDistributions.js

```diff
@@ -47,7 +47,9 @@ export function applyDistributions(
   const credIntervals = credGraph.intervals();
   const distributionIntervals = _chooseDistributionIntervals(
     credIntervals,
-    ledger.lastDistributionTimestamp(),
+    // Distributions after -Infinity are all the current
+    // distributions
+    ledger.lastDistributionTimestamp() || -Infinity,
```

Comment:
nit: this is _technically_ incorrect, since `0` is a valid timestamp corresponding to 1970, and this code would set `-Infinity` for timestamp 0. in practice, we won't have to worry about distributions from 1970, but it's good to be in a habit of not assuming that `x == null` implies `x === null` when `x` is a number. similarly when `x` is a string, since `""` is a valid string that would evaluate as false in this case.

this is what we have `NullUtil.orElse` for, recommend using that!

Reply:
That makes sense, I actually think it's really nice if there were like time travelers that used sourcecred, unironically that is lol, pushed a commit that should resolve that as well